firefox-ios/Client/Frontend/Settings/Main/AppSettingsTableViewController.swift

```diff
@@ -244,7 +244,7 @@ class AppSettingsTableViewController: SettingsTableViewController,
             prefKey: "showClipboardBar",
             defaultValue: false,
             titleText: .SettingsOfferClipboardBarTitle,
-            statusText: .SettingsOfferClipboardBarStatus
+            statusText: String(format: .SettingsOfferClipboardBarStatus, "Firefox")
```

Comment:
We should not use `Firefox` as a literal here but instead use `AppName.shortName.rawValue`.